motorway/instrumentation.py

```diff
@@ -1,18 +1,18 @@
 from contextlib import contextmanager
-
-NEW_RELIC = False
+import os
+new_relic_app = None
 try:
     import newrelic.agent
-    if newrelic.agent.application().active:
-        NEW_RELIC = True
+    if "NEW_RELIC_CONFIG_FILE" in os.environ and os.environ["NEW_RELIC_CONFIG_FILE"]:
```

Comment:
maybe if os.environ.get("NEW_RELIC_CONFIG_FILE")

Reply:
You're right